extensions/machine-learning-services/package.nls.json

```diff
@@ -9,6 +9,8 @@
 	"mls.command.mlsdocs": "Machine Learning Services Documentation",
 	"mls.configuration.title": "Machine Learning Services configurations",
 	"mls.pythonPath.description": "Local path to a preexisting python installation used by Machine Learning Services.",
-	"mls.rPath.description": "Local path to a preexisting python installation used by Machine Learning Services.",
+	"mls.enablePython.description": "Enable python package management.",
```

Comment:
shouldn't `Python` start with an uppercase character in this sentence fragment?

Reply:
yes. will change

---
In reply to: [371539291](https://github.com/microsoft/azuredatastudio/pull/8968#discussion_r371539291) [](ancestors = 371539291)